packages/salesforcedx-vscode-core/package.json

```diff
@@ -143,6 +143,18 @@
         {
           "command": "sfdx.force.source.delete.current.file",
           "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking && resourceLangId != 'forcesourcemanifest'"
+        },
+        {
+          "command": "sfdx.force.diff",
```

Comment:
I added this so we show the diff command exclusively in the supported file types/languages.

Reply:
Nice. Another option later could be to set a context based on the open file whether it's diffable or not, if you don't want the large regex.